icepyx/core/spatial.py

```diff
@@ -646,9 +729,12 @@ def fmt_for_CMR(self):
 
             cmr_extent = ",".join(map(str, extent))
 
+        else:
+            raise RuntimeError("Programmer error!")
```

Comment:
```suggestion
            raise RuntimeError("Programmer error!")
```
Note to change this to the exhaustive options error we created.